### PR TITLE
Gidk 525 ledger

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 1]
```diff
@@ -166,13 +166,13 @@
...
         <section id="eledgerwrap" class="mrT2" *ngIf="lc.showEledger">
-            <div class="alert alert-danger fade in pr" style="padding-right:15px">
+            <div class="alert alert-danger ledger-book fade in pr" style="padding-right:15px">
                 <button type="button" class="close">
                     <span style="font-size:12px;font-weight:normal"
                         (click)="lc.showEledger = !lc.showEledger">Close</span>
                 </button>
 
-                <table class="table ledgerTable" style="margin-bottom:0">
+                <table class="table ledgerTable transction-mapped-table" style="margin-bottom:0">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/ledger.component.html:175]: Avoid incorrect spelling in class names

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 2]
```diff
@@ -184,12 +184,19 @@ <h3 class="ledgerHead">
                     </thead>
                     <tbody>
                         <tr>
-                            <td class="nopad bdrL bdrR hidden-xs" style="width:50%" colspan="3">
+                            <td width="100%" colspan="100%" class="ledger-head transction-mapped">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/ledger.component.html:187]: Avoid incorrect spelling in class names

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 3]
```diff
@@ -184,12 +184,19 @@ <h3 class="ledgerHead">
...
+                                <table class="table" width="100%">
+                                    <tr>
+                                        <th colspan="6">Debit <span class="pull-right text-gray">Dr</span></th>
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/ledger.component.html:190]: Please check if new text is present in JSON else add it

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 4]
```diff
@@ -184,12 +184,19 @@ <h3 class="ledgerHead">
...
+                                        <th colspan="6">Credit <span class="pull-right text-gray">Cr</span></th>
+                                    </tr>
+                                </table>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td class="nopad bdrL bdrR hidden-xs transction-mapped-head" style="width:50%" colspan="3">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/ledger.component.html:197]: Avoid incorrect spelling in class names
Avoid inline styles

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 5]
```diff
@@ -201,13 +208,10 @@ <h3 class="ledgerHead">
                                 </table>
                             </td>
                             <!-- main left td end here -->
-                            <td class="nopad bdrL bdrR hidden-xs" style="width:50%" colspan="3">
+                            <td class="nopad bdrL bdrR hidden-xs transction-mapped-head" style="width:50%" colspan="3">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/ledger.component.html:211]: Avoid incorrect spelling in class names

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 6]
```diff
@@ -294,14 +300,18 @@ <h3 class="ledgerHead">
...
-
+                                                            <td class="col-xs-3 col-md-3 col-lg-3">
+                                                                <div class="d-flex align-items-center">
+                                                                    <span>
+                                                                        <input type="checkbox" [(ngModel)]="txn.isChecked" (click)="selectEntryForBulkAction($event, item)" />
+                                                                    </span>
+                                                                    <span class="ml-05">
+                                                                        <input class="form-control" [maxlength]="10" name="bsDatess" autocomplete="off" type="text" #cbs='bsDatepicker' [(ngModel)]="item.entryDate" bsDatepicker [bsConfig]="{ dateInputFormat: giddhDateFormat }" [outsideClick]="true">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/ledger.component.html:309]: Avoid incorrect spelling in ** bsDatess**

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html]: Done

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html]: DOne

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:190]: Added     **"app_dr": "dr."** by this name

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html]: Done
Inline css written already , i am not added this

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html]: Done

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html]: I am not added this, but i correct it

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 7]
```diff
@@ -184,12 +184,19 @@ <h3 class="ledgerHead">
                     </thead>
                     <tbody>
                         <tr>
-                            <td class="nopad bdrL bdrR hidden-xs" style="width:50%" colspan="3">
+                            <td width="100%" colspan="100%" class="ledger-head transaction-mapped">
+                                <table class="table" width="100%">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/ledger/ledger.component.html:188]: use width="100%" & use css class name

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:188]: we can use this that is attributes

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:188]: it's not a inline css

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 8]
```diff
@@ -184,12 +184,19 @@ <h3 class="ledgerHead">
...
+                                    <tr>
+                                        <th colspan="6">Debit <span class="pull-right text-gray">Dr</span></th>
+                                        <th colspan="6">Credit <span class="pull-right text-gray">Cr</span></th>
+                                    </tr>
+                                </table>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td class="nopad bdrL bdrR hidden-xs transaction-mapped-head" style="width:50%" colspan="3">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/ledger/ledger.component.html:197]: remove inline css style="width:50%"

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:197]: i am not added this css 
'

**Pull request merged**